.circleci/config.yml

```diff
@@ -20,6 +20,7 @@ executors:
     environment:
       # more CPUs visible but we're throttled to 8, which breaks auto-detect
       NUM_CPUS: 8
+      GIT_LFS_SKIP_SMUDGE: 1
```

Comment:
add comment explaining why this is needed? (I had to Google it)

Reply:
Good point. I added something to explain and a link with further details.